environments/production/terraform.yml

```diff
@@ -281,14 +281,15 @@ servers:
     volume_size: 160
     group: "kafka"
     os: trusty
-
-  - server_name: "pgproxy1-production"
-    server_instance_type: t3.large
+    
```

Comment:
nit: try to make these whitespaces a little cleaner in the future by looking over `git diff` before committing

Reply:
sure